packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -99,6 +102,13 @@ const styles = (theme: Theme) => {
         warning: {
             color: theme.palette.warning.main,
         },
+        hasHelperText: {},
+        helperText: {
+            "& $label:not([class*='Mui-focused'])": {
+                color: theme.palette.grey["300"],
+            },
```

Comment:
For this to work it would need to be in `hasHelperText`, but it isn't needed anyways.